addons/hr_skills/static/tests/many2one_avatar_employee.test.js

```diff
@@ -0,0 +1,96 @@
+import {
+    click,
+    contains,
+    openKanbanView,
+    start,
+    startServer,
+} from "@mail/../tests/mail_test_helpers";
+import { beforeEach, describe, expect, test } from "@odoo/hoot";
+import { queryAttribute } from "@odoo/hoot-dom";
+import { defineHrSkillModels } from "@hr_skills/../tests/hr_skills_test_helpers";
+
+describe.current.tags("desktop");
+defineHrSkillModels();
+
+beforeEach(async () => {
+    const pyEnv = await startServer();
+    this.data = {};
+
+    // Skills
+    [this.data.skillJavaId, this.data.skillTigrinya] = pyEnv["hr.skill"].create([
+        {
+            name: "Java",
+        },
+        {
+            name: "Tigrinya",
+        },
+    ]);
+
+    // User
+    this.data.partnerPierreId = pyEnv["res.partner"].create({
+        name: "Pierre",
+    });
+    this.data.userPierreId = pyEnv["res.users"].create({
+        name: "Pierre",
+        partner_id: this.data.partnerPierreId,
+    });
+
+    // Employees
+    this.data.employeePierreId = pyEnv["hr.employee"].create({
+        name: "Pierre",
+        user_id: this.data.userPierreId,
+        user_partner_id: this.data.partnerPierreId,
+    });
+
+    // Employee skills
+    [this.data.employeeSkill1Id, this.data.employeeSkill2Id] = pyEnv["hr.employee.skill"].create([
+        {
+            employee_id: this.data.employeePierreId,
+            skill_id: this.data.skillJavaId,
+        },
+        {
+            employee_id: this.data.employeePierreId,
+            skill_id: this.data.skillTigrinyaId,
+        },
+    ]);
+
+    // Public employee
+    // Imitating the server behavior by creating an hr.employee.public record with the same data and same id
+    pyEnv["hr.employee.public"].create({
+        name: "Pierre",
+        employee_skill_ids: [this.data.employeeSkill1Id, this.data.employeeSkill2Id],
+    });
+
+    // Fake record
+    this.data.recordPierreId = pyEnv["m2o.avatar.employee"].create([
+        { employee_id: this.data.employeePierreId },
+    ]);
+});
+
+test("many2one_avatar_employee widget in kanban view with skills on avatar card", async () => {
+    await start();
+    await openKanbanView("m2o.avatar.employee", {
```

Comment:
Do we really need mail test helpers? If not it would be better not to rely on them.